src/TrueOrFalse.Frontend.Web/Views/ApiVueApp/Components/Topic/TopicDataManager.cs

```diff
@@ -55,6 +55,7 @@ private SearchTopicItem FillMiniTopicItem(CategoryCacheItem topic)
         ImageMetaData imageMetaData,
         KnowledgeSummary knowledgeSummary)
     {
+        var sessionUser = _sessionUser.UserId == 0 ? null : _sessionUser.User;
         return new TopicDataResult(
```

Comment:
It should be visible, what vars are populated. Use something like

return new TopicDataResult(
  varName: asdfasdf, 
  ..